<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFriendItem</class>
 <widget class="QWidget" name="CFriendItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>80</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>69</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_icon">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>icon</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>11</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>name</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_signed">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>signed</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
